test/services/TxSender.test.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-empty */
```

Comment:
I'll remove this when someone suggests a better approach to test calls. See below.

Reply:
It's for the try-catch blocks.
There are some cases where the code throws an exception, but I still have to test some function calls. This rule is disabling empty blocks, such as in this case:
```js
try {
  doSomething();
} catch {}

assert(methodInsideDoSomething.called).to.be.true;
```